src/query.c

```diff
@@ -547,14 +547,40 @@ static IndexIterator *iterateExpandedTerms(QueryEvalCtx *q, Trie *terms, const c
     }
   }
 
+  // Add an iterator over the inverted index of the empty string for fuzzy search
```

Comment:
Seems like code duplication, is there a chance to create a common function to perform the addition of the iterator?

Reply:
Right 👍 
Now I remember that when I wrote I said I'll make a function, and forgot.